build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -49,12 +54,13 @@ public override string ToString()
         //
 
         // Full versions and package information.
+        public bool EnsureStableVersion => true;
```

Comment:
I don't think we should turn this on until we are 99.99% sure we aren't going to spin a new build of the DotNetHost